lib/rules/web/ocm_console/cluster_detail_machinepool_tab.xyaml

```diff
@@ -121,4 +123,97 @@ click_scale_machine_pool_button:
     element:
         selector:
             xpath: //li/button[text()='Scale']
-        op: click
\ No newline at end of file
+        op: click
+check_edit_taints_in_dropdown:
+    action: expand_actions_of_machine_pool
+    element:
+        selector:
+            xpath: //button[text()='Edit taints']
+click_edit_taints_in_dropdown:
+    element:
+        selector:
+            xpath: //button[text()='Edit taints']
+        op: click
+check_no_default_in_dropdown_when_editing_taints:
+    elements:
+        - selector:
+            xpath: //select[@id='machinePoolId']
+          op: click
+        - selector:
+            xpath: //option[text()='Default']
+          missing: true
+check_for_taints_row:
+    elements:
+        - selector:
+            xpath: //input[@id='taints[<row_num>].key']
+        - selector:
+            xpath: //input[@id='taints[<row_num>].value']
+        - selector:
+            xpath: //select[@id='taints[<row_num>].effect']
+check_edit_taints_dialog:
```

Comment:
All of the actions packaged for dialog should be moved to file lib/rules/web/ocm_console/dialogs_and_sidebar.xyaml